HPXMLtoOpenStudio/resources/EPvalidator.stron

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sch:schema xmlns:sch="http://purl.oclc.org/dsdl/schematron">
+  <sch:title>HPXML Schematron Validator</sch:title>
+  <sch:ns uri='http://hpxmlonline.com/2019/10' prefix='hpxml_v3'/>
+  <sch:pattern>
+    <sch:title>EnergyPlus Validator</sch:title>
+    <sch:rule context="/hpxml_v3:HPXML/hpxml_v3:XMLTransactionHeaderInformation">
+      <sch:assert test="count(hpxml_v3:XMLType) = 1">element 'hpxml_v3:XMLType' is REQUIRED</sch:assert>
```

Comment:
Hmm, I kind of expected to just see "element 'XMLType' is REQUIRED" instead of "element 'hpxml_v3:XMLType' is REQUIRED". Not sure what the pros/cons are.

Reply:
The former is likely to be easier to understand.

Reply:
There's only a single namespace in HPXML though, right? So I don't really see what additional information is being provided. I would guess that most users/developers are largely unaware that there's even a namespace. To that point, our current EPvalidator does not provide error messages that include the namespace.

Aside from the BuildingSync example, I'm not really finding any examples where the namespace is included in the error messages. I am leaning towards thinking we shouldn't include them.